lottie/src/main/java/com/airbnb/lottie/animation/keyframe/PathKeyframeAnimation.java

```diff
@@ -20,11 +20,12 @@ public PathKeyframeAnimation(List<? extends Keyframe<PointF>> keyframes) {
   }
 
   @Override public PointF getValue(Keyframe<PointF> keyframe, float keyframeProgress) {
+    if (keyframe.startValue == null || keyframe.endValue == null) {
```

Comment:
Is this strictly necessary? I'm afraid that this might cause backwards compatibility issues. Companies sometimes try to minify their animations in unexpected ways and I could imagine this crashing for animations that used to work.